app/renderer/js/utils/link-util.ts

```diff
@@ -22,12 +22,12 @@ class LinkUtil {
 	}
 
 	isInternal(currentUrl: string, newUrl: string): IsInternalResponse {
-		const currentDomain = wurl('hostname', currentUrl);
-		const newDomain = wurl('hostname', newUrl);
+		const currentDomain: string = wurl('hostname', currentUrl);
```

Comment:
Explicitly mentioning type is not required anywhere. Same for all 4 changes below.

Reply:
A doubt here actually! When (in VS Code), I hover to these vars (currentDomain and others), they say :any. So isn't an explicit mention required to avoid the type 'any' ?

Reply:
No, not required.